### PR TITLE
Sphinx manual

### LINKED ISSUE #113: update documentation regarding windows installation
It says "windows is not supported". But there is a *_setup.exe file.

So some minimal installation instruction should be provided.

### LINKED ISSUE #504: Documentation in docs/technical.rst contains inconsistent duplicated flairs.
"Pypi" and "docs" flairs are duplicated above the overview table. The contents of the "Pypi" flair are inconsistent between occurrences.

image

### LINKED ISSUE #552: Generate nice documentation using sphinx and publish to github pages

### LINKED ISSUE #564: Sphinx build failing
Sphinx build fails in GithubActions:

https://github.com/pytrip/pytripgui/runs/4728409134?check_suite_focus=true
image

### PR DESCRIPTION
**Pull request by michal367**
closes #564 
closes #113 
closes #552 
closes #504 

- #564 
- #113 
- #552 
- #504

To view created site download artifact `html-docs` available here
https://github.com/pytrip/pytripgui/actions/runs/1671932226

### CODE REVIEW

File: docs/usage.rst [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+TRIP98 can be set up either on your local machine or a remote one, so choose the one that suits your needs.
+Presented below are both of the configuration views with explanations for each field.
+
+.. image:: _images/trip_configuration_local.png
+	:width: 49 %
+
+.. image:: _images/trip_configuration_remote.png
+	:width: 49 %
+
+Next up is the beam kernel setup. From the main application view, click on *Settins* and *Beam Kernels*. Below you can find its configuration view.
```
Comment (Yoanka) [inline docs/usage.rst:24]:
```suggestion
Next up is the beam kernel setup. From the main application view, click on *Settings* and *Beam Kernels*. Below you can find its configuration view.
```

File: docs/usage.rst [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+.. image:: _images/beam_kernel_setup.png
+	:align: center
+	:width: 65 %
+
+
+Main usage scenario
+===================
+
+In this usage scenario you will import some patient data either from a DICOM or a Voxelplan, configure a simulation of a treatment plan, execute it and observe its results.
```
Comment (Yoanka) [inline docs/usage.rst:37]:
```suggestion
In this usage scenario you will import patient data either from DICOM or Voxelplan files, configure a simulation of a treatment plan, execute it and observe its results.
```

File: docs/usage.rst [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+.. image:: _images/new_patient_smol.png
+	:align: center
+	
+After the selected data gets loaded in, you should be presented with the following dialog box:
+
+.. image:: _images/contour_request.png
+	:align: center
+	
+Regardless of whether you choose to precalculate VOI contours or not, a position with the imported data should be visible under *Patient tree*, on the left.
```
Comment (Yoanka) [inline docs/usage.rst:49]:
```suggestion
Regardless of whether you choose to precalculate VOI contours or not, an entry with the imported data should be visible under *Patient tree*, on the left.
```

File: docs/usage.rst [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+Here you can switch between imported files, simply by clicking on their name. You should also see a preview of the imported data, more specifically one of its slices, which you can
```
Comment (Yoanka) [inline docs/usage.rst:50]:
```suggestion
Here you can switch between imported files by clicking on their name. You should also see a preview of the imported data, more specifically one of its slices, which you can
```

File: docs/usage.rst [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+change by using the scroll wheel or moving the slider visible on the far right. Above the preview, you can also switch between the three different planes with a drop-down list.
```
Comment (Yoanka) [inline docs/usage.rst:51]:
```suggestion
switch by using the scroll wheel or moving the slider visible on the far right. Above the preview, you can also switch between the three different planes with a drop-down list.
```

File: docs/usage.rst [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+Below *Patient tree*, you can find *VOI list* from which you can select any VOIs that you'd like to make visible and see in the preview.
```
Comment (Yoanka) [inline docs/usage.rst:52]:
```suggestion
Below the *Patient tree*, you can find the *VOI list* from which you can select any VOIs that you'd like to make visible and see its contours in the preview.
```

File: docs/usage.rst [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+.. image:: _images/main_window_patient_loaded_contours.png
+	:align: center
+	
+To configure the treatment plan, click on *Create plan*, go to the *Target* tab and then select one of the positions from *Target ROI*.
```
Comment (Yoanka) [inline docs/usage.rst:57]:
```suggestion
To configure the treatment plan, click on *Create plan*, go to the *Target* tab and then select one of the entries from *Target ROI*.
```

File: docs/usage.rst [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+	:align: center
+	
+Note that the created treatment plans and fields are also visible under *Patient tree*.
+
+Finally, to execute the simulation of the created treatment plan, click on *Execute*. You should see a window pop up, which reports on the progress of the simulation.
+
+.. image:: _images/executing_plan_start.png
+	:align: center
+	
+The simulation completes soon after the **Done** message is displayed. You can click *OK* to close the window.
```
Comment (Yoanka) [inline docs/usage.rst:79]:
```suggestion
The simulation completes soon after the *Done* message is displayed. You can click *OK* to close the window.
```
All quotes in italics for consistency.

File: docs/usage.rst [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+.. image:: _images/executing_plan_end_wo_dvh.png
+	:align: center
+	
+You should be able to see the distribution of the calculated dose in the preview. If the results also include a LET distribution you can switch to it by expanding the simulation position
```
Comment (Yoanka) [inline docs/usage.rst:84]:
```suggestion
You should be able to see the distribution of the calculated dose in the preview. If the results also include a LET distribution you can switch to it by expanding the simulation entry
```

File: docs/usage.rst [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+from under *Patient tree* and clicking on the appropriate item.
```
Comment (Yoanka) [inline docs/usage.rst:85]:
```suggestion
under *Patient tree* and clicking on the appropriate item.
```

**Review by Yoanka — COMMENTED** [review]
Some wording suggestions.

**Review by Yoanka — COMMENTED** [review]
More wording suggestions for the user guide.

File: docs/user_guide.rst [hunk 11]
```diff
@@ -0,0 +1,208 @@
...
+.. image:: _images/main_window_export.png
+	:align: center
+
+|
+|
+
+Import dose data
+----------------
+
+You can import dose data (part of simulation results) that you previously exported, onto patient data, by right clicking
```
[the remark this answers is not in the document]
Comment (Hewwho) [reply docs/user_guide.rst:203]: Dose data in general, not some specific one, so without "the".
No need for past perfect here.

**Review by Hewwho — APPROVED** [review]
LGTM

**Review by Yoanka — APPROVED** [review]
LGTM

**Review by grzanka — APPROVED** [review]
LGTM

**Pull request merged**